hydra-node/src/Hydra/Chain.hs

```diff
@@ -185,7 +191,10 @@ data Chain tx m = Chain
   -- reasonable local view of the chain and throw an exception when invalid.
   --
   -- Does at least throw 'PostTxError'.
-  , draftTx :: (IsChainState tx, MonadThrow m) => UTxOType tx -> m (Either (PostTxError tx) tx)
+  , draftTx ::
```

Comment:
Should: name this `draftCommitTx` as it is only about commit transactions right now.